sys/include/xtimer/tick_conversion.h

```diff
@@ -29,7 +29,78 @@ extern "C" {
 #endif
 
 /* Some optimizations for common timer frequencies */
+/*
+ * Adaptation for frequencies multiple of 921600, usually used to match
+ * baud rates on serial communications
+ */
+#if (XTIMER_HZ % 576 == 0)
 #if (XTIMER_SHIFT != 0)
+#if (XTIMER_HZ % 576 != 0)
```

Comment:
Yes, I can remove it but I need a way to throw the error if shifting is enabled.